demisto_sdk/commands/update_release_notes/tests/update_rn_test.py

```diff
@@ -0,0 +1,232 @@
+        version_number = update_rn.bump_version_number(pre_release=False)
+        assert version_number == expected_version
+        os.remove(os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/_pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack/pack_metadata.json'))
+
+    def test_bump_version_number_revision_overflow(self):
+        """
+            Given:
+                - a pack name and invalid version
+            When:
+                - bumping the version number in the metadata.json
+            Then:
+                - return ValueError
+        """
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/_pack_metadata.json'))
+        from demisto_sdk.commands.update_release_notes.update_rn import UpdateRN
+        update_rn = UpdateRN(pack="HelloWorld", update_type='revision', pack_files={'HelloWorld'})
+        update_rn.metadata_path = os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json')
+        self.assertRaises(ValueError, update_rn.bump_version_number)
+        os.remove(os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json'))
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/_pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json'))
+
+    def test_bump_version_number_minor_overflow(self):
+        """
+            Given:
+                - a pack name and invalid version
+            When:
+                - bumping the version number in the metadata.json
+            Then:
+                - return ValueError
+        """
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/_pack_metadata.json'))
+        from demisto_sdk.commands.update_release_notes.update_rn import UpdateRN
+        update_rn = UpdateRN(pack="HelloWorld", update_type='minor', pack_files={'HelloWorld'})
+        update_rn.metadata_path = os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json')
+        self.assertRaises(ValueError, update_rn.bump_version_number)
+        os.remove(os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json'))
+        shutil.copy(src=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/_pack_metadata.json'),
+                    dst=os.path.join(TestRNUpdate.FILES_PATH, 'fake_pack_invalid/pack_metadata.json'))
+
+    def test_bump_version_number_major_overflow(self):
+        """
+            Given:
+                - a pack name and invalid version
```

Comment:
Is it in valid or is it valid and we just get overflow?
fix `Given` and `When` accordingly be specific.